tests/array/sample.js

```diff
@@ -0,0 +1,19 @@
+var chai = require('chai');
+var expect = chai.expect;
+
+require('../../array/sample');
+
+describe('sample', function() {
+  it('should grab random value from array and return it', function() {
+    var array = [1,2,3,4,5,6,7,8,9];
+
+    expect(typeof array.sample()).to.eql("number");
```

Comment:
Hmmmm. How about you test that the returned array has only one element, and that the element was in the original array.